catalog/app/containers/Bucket/PackageDialog/DialogError.js

```diff
@@ -86,7 +87,7 @@ export default function DialogError({ error, skeletonElement, title, onCancel })
     <>
       <M.DialogTitle>{title}</M.DialogTitle>
 
-      <M.DialogContent className={classes.content}>
+      <M.DialogContent style={dialogContentStyles}>
```

Comment:
actually, `className` _is_ supported, tho undocumented: https://github.com/mui-org/material-ui/blob/v4.11.3/packages/material-ui/src/DialogContent/DialogContent.js#L36 (`classes.root` is supported and documented)

Reply:
So, I'll use `className` then, if you don't mind?